Headers/Foundation/NSGeometry.h

```diff
@@ -97,8 +97,8 @@ typedef NSSize *NSSizePointer;
 }</example>
 
  <p>Rectangle.</p> */
-typedef struct _NSRect NSRect;
-struct _NSRect
```

Comment:
Since we’re now defining `struct CGPoint/CGSize/CGRect`, can we please also add the following typedefs to match `CFCGTypes.h` from Apple’s CoreFoundation:

```c
typedef struct CGPoint CGPoint;
typedef struct CGSize CGSize;
typedef struct CGRect CGRect;
```